dockstore-integration-testing/src/test/resources/integrationtest.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDDR+DxQI1yXcTX
```

Comment:
Uh oh, legit or not? 
If not, this should be filled with more obviously garbage content, like the script for a reality show or something.

Reply:
I generated it locally for a self-signed cert that is not being used anywhere else, so nothing compromised. I think the code is fairly picky about the contents being valid, but will try to indicate that it's not real.

Reply:
After some experimenting, the contents have to be valid, or the code that reads the key throws an exception.

When I squash and merge, I'll update the commit message describe the origin of this file.

Reply:
In that case, it looks like you can add a comment to the file 
https://stackoverflow.com/questions/19578812/comments-in-a-pem-file
Looks like there was a bug that was fixed for comment parsing in Java in Java 6, which implies it should be working now in 11 
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=539283

Reply:
Looks like [our code](https://github.com/dockstore/dockstore/blob/f10d8300ad3bb2434bf9e7c4893fed601d522a85/dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubHelper.java#L207) that reads and processes the PEM doesn't ignore comments. :(

Do you think it's worth updating that code to handle comments?

Reply:
Yeah, or see if there is a pre-existing library method as that bug report would lead me to believe there is.

Reply:
Fixed in upcoming push